src/blocks/woo-portal/search/assets/scripts/vue/Woo.vue

```diff
@@ -12,42 +12,21 @@
     <!-- Verzoek + Besluit -->
     <div class="woo-portal-detail__columns">
       <div class="woo-portal-detail__column">
-        <h2 class="woo-portal-detail__subtitle">Verzoek</h2>
+        <h2 class="woo-portal-detail__subtitle">Ontvangstdatum</h2>
         <time
             v-if="detail.Ontvangstdatum"
             v-html="dateReadable(detail.Ontvangstdatum)"
             :datetime="detail.Ontvangstdatum"
         />
-        <a
-            v-if="detail.URL_informatieverzoek"
-            :href="detail.URL_informatieverzoek"
-            class="btn"
-            target="_blank"
-        >
-          Bekijk informatieverzoek in PDF
-        </a>
-
-<!--        <button v-if="!detail.URL_informatieverzoek" class="btn" disabled>-->
-<!--          Bekijk informatieverzoek in PDF-->
-<!--        </button>-->
       </div>
 
       <div class="woo-portal-detail__column">
-        <h2 class="woo-portal-detail__subtitle">Besluit</h2>
+        <h2 class="woo-portal-detail__subtitle">Besluitdatum (date)</h2>
         <time
```

Comment:
`(date)` dubbelop?

Reply:
Can u check by design? Looks like a mistake to me if i read it :P 

I read it as "DATE DATE"

Reply:
<img width="793" alt="Screenshot 2024-01-16 at 17 30 51" src="https://github.com/OpenWebconcept/plugin-woo-portal-plugin/assets/107254793/17a305c2-1bc0-431c-884e-9d08fd021aa1">

Reply:
Looks like a design flaw to me than xD